desktop/cmp/input/NumberInput.ts

```diff
@@ -226,6 +226,9 @@ class NumberInputModel extends HoistInputModel {
     }
 }
 
+// Note: we don't use the `ref` here, but the presence of a second argument is required
```

Comment:
Should be one-line comment at most, especially since repeated.  Does typescript complain if we were to take it away?

Reply:
No complaint from TS - just didn't want someone to see the unused second argument and remove it assuming it's cruft.